<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cuDNNv</class>
 <widget class="QMainWindow" name="cuDNNv">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>120</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>cuDNNv: cuDNN Version Switcher</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="font">
    <font>
     <family>NanumGothic</family>
    </font>
   </property>
   <widget class="QLabel" name="versionLabel">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>100</y>
      <width>90</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>NanumMyeongjo</family>
      <pointsize>9</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>cuDNNv v0.4</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>200</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>NanumMyeongjo</family>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="switchButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>90</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>NanumMyeongjo</family>
     </font>
    </property>
    <property name="text">
     <string>Switch</string>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>55</y>
      <width>390</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>NanumMyeongjo</family>
     </font>
    </property>
    <property name="text">
     <string>Choose the version you want to switch to</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
